testMVVM/TabBarViewController.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  TabBarViewController.swift
+//  MVVMTestProject
+//
+//  Created by Petr Zvoníček on 06.10.16.
+//  Copyright © 2016 Martin Pinka. All rights reserved.
+//
+
+import UIKit
+
+class TabBarViewController: UITabBarController {
```

Comment:
This file is not used and named view controller even though it is only controller.